packages/web-components/src/components/va-textarea/test/va-textarea.e2e.ts

```diff
@@ -12,11 +12,11 @@ describe('va-textarea', () => {
     expect(element).toEqualHtml(`
       <va-textarea class="hydrated" label="Describe your situation">
         <mock:shadow-root>
-          <label for="textarea">
+          <label for="textarea" part="label">
```

Comment:
Is there a specific reason that we need to override the label style?

Reply:
I don't have a specific reason myself. I included the "part" attribute on the `label` and `small` (validation) elements so that the textarea component had the same level of styling control as the va-text-input component. https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/web-components/src/components/va-text-input/va-text-input.tsx

Reply:
@jamigibbs If you think the part attribute on the label tag is ill-advised or unnecessary, I can remove it. Just let me know. Thanks for reviewing this for me!

Reply:
@mattwrightva  Thanks for your patience and we appreciate you creating a PR too.

We were discussing internally as a team the most efficient way to allow for custom styling while also maintaining some control over those elements as part of a design system. With that said, for now I think we'd like to only add the `part` attribute to elements that have specific blockers for teams right now and if other specific needs arise later (like label here), we can consider adding it then.

Do you mind updating this PR to only include `part` on the textarea element that you need for now?

Reply:
@jamigibbs No problem! I removed it.